src/logic/messages/EventManager.cpp

```diff
@@ -105,6 +119,43 @@ void EventManager::processMessageQueue()
     if(m_EventQueue.empty())
         return;
 
+    bool print_queue = false;
+    if (print_queue){
+        bool dialog_ready_to_play = false;
+        for(EventMessages::EventMessage* ev : m_EventQueue)
+        {
+            if (ev->messageType == EventMessages::EventMessageType::Conversation)
+            {
+                EventMessages::ConversationMessage* conv_message = reinterpret_cast<EventMessages::ConversationMessage*>(ev);
+                if (!conv_message->internInProgress && conv_message->text != std::string(""))
+                {
+                    dialog_ready_to_play = true;
+                    break;
+                }
+            }
+            if(!ev->isOverlay)
+                break;
+        }
+        if (dialog_ready_to_play){
+            LogInfo() << "----------------------Queue-Start----------------------";
+            for(EventMessages::EventMessage* ev : m_EventQueue)
+            {
+                if (ev->messageType == EventMessages::EventMessageType::Conversation)
+                {
+                    EventMessages::ConversationMessage* conv_message = reinterpret_cast<EventMessages::ConversationMessage*>(ev);
+                    if (!conv_message->internInProgress && conv_message->text != std::string(""))
+                    {
+                        LogInfo() << "Queue item: " << std::boolalpha << "block = " << !ev->isOverlay << ", text = " << conv_message->text;
+                    } else if (conv_message->subType == EventMessages::ConversationMessage::ST_WaitTillEnd){
+                        auto waiting_for = reinterpret_cast<const EventMessages::ConversationMessage*>(conv_message->waitIdentifier);
+                        LogInfo() << "Queue item: " << std::boolalpha << "block = " << !ev->isOverlay << ", waiting for: " << conv_message << ", ident = " << waiting_for->text;
+                    }
+                }
+            }
+            LogInfo() << "----------------------Queue-End----------------------";
+        }
+    }
+
```

Comment:
If this code is for debugging purposes only, remember to remove it before merge please.